### ISSUE TITLE
rv-each loop not working !

### ISSUE THREAD

**Issue by nsisodiya** [mikeric/rivets#438]
Here is my Data 
```
var p = {
person : ["a", "b", "c"]
}
```
which i want to show in html using ```rv-each``` binding. It was working with old version of rivets.js but now with latest release it stopped working.
May someone give me working code on jsfiddle etc.

**Comment by mikeric** [comment]
Did you even try this before opening an issue? If the problem is only occurring in your own application code, then you'll need to at least attempt to provide an isolated example of what is "not working".

> May someone give me working code on jsfiddle etc.

http://jsfiddle.net/q8w95b7v/

**Comment by nsisodiya** [comment]
@mikeric -
I copied a working work and added a rv-each loop in my code which was not working.
Initially I thought that the error happened due to version update in my code - 0.7.1 to 0.7.2.
After lots of debugging, I found my silly mistake !

```
<ul>
    <li rv-each-person="scope.persons">{ person }</li>
</ul>
```
The above rv-each code was pasted outside the main div on which I was adding rivets binding.
So it was my silly mistake.

Sorry !
